webpack.config.js

```diff
@@ -50,8 +50,9 @@ module.exports = env => {
           jQuery: 'jquery',
           'window.jQuery': 'jquery',
         }),
-        new ExtractTextPlugin({
-          filename: '[name].[contenthash].min.css',
+        new MiniCssExtractPlugin({
+          filename: '[name].[contenthash].scss',
```

Comment:
Is this needed? An object with 2 identical keys will use the second one. Also I don't think we want to output `scss`.